.github/workflows/forest.yml

```diff
@@ -21,8 +21,8 @@ env:
   AWS_ACCESS_KEY_ID: '${{ secrets.AWS_ACCESS_KEY_ID }}'
   AWS_SECRET_ACCESS_KEY: '${{ secrets.AWS_SECRET_ACCESS_KEY }}'
   RUSTC_WRAPPER: sccache
-  CC: sccache clang
```

Comment:
Do we have to switch to gcc? clang should be working without the apt step as it's preinstalled in the runner image
https://github.com/actions/runner-images/blob/main/images/ubuntu/Ubuntu2204-Readme.md#language-and-runtime